sys/shell/commands/sc_netif.c

```diff
@@ -36,6 +36,7 @@
 #include "net/gnrc/pktbuf.h"
 #include "net/gnrc/netif/hdr.h"
 #include "net/gnrc/sixlowpan/netif.h"
+#include "net/netdev2/ieee802154.h"
```

Comment:
Don't like how this includes IEEE 802.15.4 depending code. It invokes memories from `transceiver` times. What if the interface isn't an IEEE 802.15.4 device? Are there devices that have different CCA modes, but are not IEEE 802.15.4?